src/app/zap-templates/zcl/data-model/manufacturers.xml

```diff
@@ -177,6 +177,7 @@ Note:
   <mapping code="0x1097" translation="SyChip/Murata"/>
   <mapping code="0x1098" translation="OpenPeak"/>
   <mapping code="0x1099" translation="PassiveSystems"/>
+  <mapping code="0x109B" translation="Pioneer Corp"/>
```

Comment:
@markus-becker-tridonic-com The 0x109B is now duplicated ... WHich one is correct?

Reply:
@Apollon77 I don't know. They are both listed in the PDF.

Reply:
But should it then not better ONE entry in the List with a "combined name" instead of the list getting a duplicated code? ... On the other side then the question is if that PDF is wrong ;-)

Reply:
I am discussing with the author of the PDF from CSA on Slack to clarify which is the correct entry.

Reply:
Solved in https://github.com/project-chip/connectedhomeip/pull/30483.

Reply:
Thank you very much for clarification!